src/main/java/andersen/lesson/validator/Validator.java

```diff
@@ -0,0 +1,20 @@
+package andersen.lesson.validator;
+
+import java.math.BigDecimal;
+
+public class Validator {
+
+    public BigDecimal validateByPrice(BigDecimal price) {
+        if (price.compareTo(new BigDecimal(0)) < 0) {
```

Comment:
There is special constant for that BigDecimal.ZERO you should use it instead of new BigDecimal(0)